### PR TITLE
[QA-244]: Updated IPV Core script

### PR DESCRIPTION
**Pull request by abhinaysrivastavaperf**
### What?
Updated IPV core script

#### Changes:
- Added the steps for DCMAW stub
- Updated page content and group names wherever applicable

---

### Why?
Due to the latest changes in the IPV core user journey with orchestrator stubs in build environment

---

### Related:
- [IPV Core happy path scenario](https://govukverify.atlassian.net/wiki/spaces/DID/pages/3346661421/Performance+Test+Scenarios#IPV-Core-Full-Journey---Build-Environment-Using-Stubs)

### CODE REVIEW

File: deploy/scripts/src/ipv-core/test.ts [hunk 1]
```diff
@@ -294,12 +367,13 @@ export function coreScenario1 (): void {
...
 
   group(
-    'B01_Core_06_AddrDataContinue POST',
+    'B01_Core_07_AddrDataContinue POST',
     function () {
       const startTime1 = Date.now()
       res = http.post(
         addressStubURL,
         {
+          sort: 'Kenneth Decerqueira (Valid Experian) Address',
```
Comment (Tom-Dann) [inline deploy/scripts/src/ipv-core/test.ts:376]:
```suggestion
          sort: user.address,
```
Suggestion: Paramaterise these values

File: deploy/scripts/src/ipv-core/test.ts [hunk 2]
```diff
@@ -462,12 +538,13 @@ export function coreScenario1 (): void {
...
 
   group(
-    'B01_Core_09_KBVDataContinue POST',
+    'B01_Core_10_KBVDataContinue POST',
     function () {
       const startTime1 = Date.now()
       res = http.post(
         kbvStubURL,
         {
+          sort: 'Kenneth Decerqueira (Valid Experian) KBV',
```
Comment (Tom-Dann) [inline deploy/scripts/src/ipv-core/test.ts:547]:
```suggestion
          sort: user.kbv,
```
Suggestion: Paramaterise these values

File: deploy/scripts/src/ipv-core/test.ts [hunk 3]
```diff
@@ -222,12 +294,13 @@ export function coreScenario1 (): void {
...
 
   group(
-    'B01_Core_05_PassportDataContinue POST',
+    'B01_Core_06_PassportDataContinue POST',
     function () {
       const startTime1 = Date.now()
       res = http.post(
         passportStubURL,
         {
+          sort: 'Kenneth Decerqueira (Valid Experian) Passport',
```
Comment (Tom-Dann) [inline deploy/scripts/src/ipv-core/test.ts:303]:
```suggestion
          sort: user.passport,
```
Suggestion: Paramaterise these values

File: deploy/scripts/src/ipv-core/test.ts [hunk 4]
```diff
@@ -148,25 +146,99 @@ export function coreScenario1 (): void {
...
+      })
+        ? transactionDuration.add(endTime1 - startTime1)
+        : fail('Response Validation Failed')
+
+      dcmawStubURL = res.headers.Location
+
+      const startTime2 = Date.now()
+      res = http.get(res.headers.Location,
+        {
+          tags: { name: 'B01_Core_03_ClickStartAfterLogin_02_DcmawStub' }
```
Comment (Tom-Dann) [inline deploy/scripts/src/ipv-core/test.ts:175]: Simplify this tag or add `// pragma: allowlist secret`

Comment (abhinaysrivastavaperf) [reply deploy/scripts/src/ipv-core/test.ts:175]: This is done in the next commit

File: deploy/scripts/src/ipv-core/test.ts (hunk 1 above)
Comment (abhinaysrivastavaperf) [reply deploy/scripts/src/ipv-core/test.ts]: Done

File: deploy/scripts/src/ipv-core/test.ts (hunk 2 above)
Comment (abhinaysrivastavaperf) [reply deploy/scripts/src/ipv-core/test.ts]: Done

File: deploy/scripts/src/ipv-core/test.ts (hunk 3 above)
Comment (abhinaysrivastavaperf) [reply deploy/scripts/src/ipv-core/test.ts]: Done

File: deploy/scripts/src/ipv-core/test.ts [hunk 5]
```diff
@@ -363,14 +437,16 @@ export function coreScenario1 (): void {
...
 
   group(
-    'B01_Core_07_FraudDataContinue POST',
+    'B01_Core_08_FraudDataContinue POST',
     function () {
       const startTime1 = Date.now()
       res = http.post(
         fraudStubURL,
         {
+          sort: 'Kenneth Decerqueira (Valid Experian) Fraud',
```
[the remark this answers is not in the document]
Comment (abhinaysrivastavaperf) [reply deploy/scripts/src/ipv-core/test.ts]: Done

**Review by Tom-Dann — APPROVED** [review]

**Pull request merged**